README.md

```diff
@@ -0,0 +1,73 @@
+# Sturdy Lamp 💡
+
+- [Overview](#overview)
+- [Features](#features)
+- [Technologies](#technologies)
+- [Local setup](#local-setup)
+- [Project workflow](#project-workflow)
+- [Authors](#authors)
+
+## Overview
+
+**What is it?** A forum where users can post their opinions.
```

Comment:
don't know what else to call it 🙈

Reply:
@Skidle We can just write a simple text instead: "Sturdy Lamp is a forum where users can freely express their opinions. This project was created as a learning playground to help authors deepen their knowledge in web technologies."

or 

'**Project description:**A forum where users can post their opinions.
**Project purpose:** This project serves as a learning playground to deepen authors' knowledge in web technologies."

Reply:
will do in a follow-up https://github.com/metisova/sturdy-lamp/issues/26